shopify-features/models/profiles.yaml

```diff
@@ -77,12 +86,45 @@ models:
 
         - entity_var:
             name: max_timestamp_bw_tracks_pages
-            select: CASE WHEN max_timestamp_tracks>=max_timestamp_pages THEN max_timestamp_tracks ELSE max_timestamp_pages END
+            select: greatest(coalesce(max_timestamp_tracks,max_timestamp_pages),coalesce(max_timestamp_pages,max_timestamp_tracks))
         #days since last seen
         - entity_var:
             name: days_since_last_seen
-            select: "{% if !(end_time|isnil) %} datediff(day, date(max_timestamp_bw_tracks_pages),date('{{end_time.Format(\"2006-01-02 15:04:05\")}}')) {% else %} 1=1 {% endif %}"
-#       #Churn features
+            # select: datediff(day, date(max_timestamp_bw_tracks_pages),date('{{timestamp.Format("2006-01-02 15:04:05")}}'))
+            select: "{{macro_datediff('max_timestamp_bw_tracks_pages')}}"
+   
+        #avg_units_per_transaction
+        - entity_var:
+            name: total_units_across_transactions
```

Comment:
These (total_units_across_transactions, total_transactions_units etc) were missing in the existing main branch? Were they deleted by us earlier?

Reply:
Yeah. I think they were deleted.